readme.md

```diff
@@ -87,7 +87,14 @@ Note that if you set this option to `false`, you won't get back matched director
 Type: `boolean`\
 Default: `false`
 
-Respect ignore patterns in `.gitignore` files that apply to the globbed files.
+Respect ignore patterns in `.gitignore` files that apply to the globbed files. This option takes precedence over the `ignoreFiles` option, and explicitly looks for files using a `**/.gitignore` glob.
+
+##### ignoreFiles
+
+Type: `string`\
+Default: `undefined`
+
+A glob pattern to look for ignore files, which are then used to ignore globbed files. This is a more generic form of the `gitignore` option, allowing you to find ignore files with a [compatible syntax](http://git-scm.com/docs/gitignore). For instance, this is works with Babel's `.babelignore`, Prettier's `.prettierignore`, or ESLint's `.eslintignore` files.
```

Comment:
Readme docs and index.d.ts docs should be in sync.

Reply:
Synced.